tst/testextra/perfect.tst

```diff
@@ -10,4 +10,16 @@ gap> Length(l);
 gap> l:=Practice(10752);;
 gap> Length(l);
 9
+gap> LoadPackage("atlasrep");
```

Comment:
This line of the test fails. It just needs a second semi colon:
`gap> LoadPackage("atlasrep");;`

Reply:
@jackschmidt you can suggest a specific change by clicking this button:
<img width="683" alt="Screenshot 2021-06-15 at 12 21 22" src="https://user-images.githubusercontent.com/8956868/122044292-3f147c00-cdd4-11eb-9d75-3f0345d818f8.png">
Which gives you some text that you edit to look like this:
<img width="194" alt="Screenshot 2021-06-15 at 12 28 33" src="https://user-images.githubusercontent.com/8956868/122045114-3b352980-cdd5-11eb-84d9-5a7e87d52e17.png">

Once you've posted a comment with a specific change, you can then click a "Commit suggestion" button somewhere on the comment to create a new commit with that change, which is pushed onto the branch that the PR is made from.

You can also click here in the top-right of any file in the "Files changed" tab of a pull request and edit the file in question there, too (although in a not-very-nice plain-text interface, in my opinion). This then saves your changes into a new commit on the PR branch:
<img width="177" alt="Screenshot 2021-06-15 at 12 22 13" src="https://user-images.githubusercontent.com/8956868/122044494-7be07300-cdd4-11eb-88cf-bcabc306b158.png">

Alternatively, you can simply push a commit to the PR branch from your local computer (i.e. in this case you could push directly to the fixes branch on @hulpke's own repository).

(If these options are available to you, you might need some GitHub permissions for `gap-system`, which would be happily given to you - but I don't know the details of this).

We normally only do the first of these three options here for reasons of etiquette, and so that the PR author can decide which changes to accept, and how to arrange the commit history of their PR.